dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -518,18 +522,19 @@ def _valid_s3_bucket_arn(key):
             }
         )
 
-    @staticmethod
-    def __s3_bucket_schema() -> dict:
-        return PlatformConfigSchema.__valid_s3_base_definition() | {
-            "type": "s3",
-            Optional("objects"): [
-                {"key": str, Optional("body"): str, Optional("content_type"): str}
-            ],
-        }
-
     @staticmethod
     def __s3_bucket_policy_schema() -> dict:
-        return PlatformConfigSchema.__valid_s3_base_definition() | {"type": "s3-policy"}
+        return dict(
+            {
+                "type": "s3-policy",
```

Comment:
Is this `s3-policy` different to the `s3-bucket-policy` as mentioned in the [docs](https://platform.readme.trade.gov.uk/reference/platform-config-yml/#s3-bucket-policy:~:text=Document%20configuration%20options.-,S3%20bucket%20policy,-platform%2Dconfig.yml)

Reply:
The keys are wrong in the dos there, but that the next thing I'm going to be doing.